charts/kof-storage/files/dashboards/opencost-cost-reporter-detailed-overview.yaml

```
@@ -2138,6 +2139,21 @@ templating:
       tagsQuery: ""
       type: query
       useTags: false
+    - datasource:
+        type: {{ $defaultDatasource }}
+        uid: "${datasource}"
+      {{ if gt (len $Values.grafana.dashboard.filters) 0 }}
+      filters:
+      {{ range $key, $value := $Values.grafana.dashboard.filters }}
+        - key: {{ $key }}
+          operator: "="
+          value: {{ $value }}
+      {{ end }}
+      {{ end }}
```

Simpler:
```suggestion
      {{ with $Values.grafana.dashboard.filters }}
      filters:
        {{ range $key, $value := . }}
        - key: {{ $key }}
          operator: "="
          value: {{ $value }}
        {{ end }}
      {{ end }}
```

From https://pkg.go.dev/text/template#hdr-Actions
> `{{with pipeline}} T1 {{end}}`
> If the value of the pipeline is empty, no output is generated;
> otherwise, dot is set to the value of the pipeline and T1 is executed.